src/main/resources/com/google/api/codegen/ruby/method_sample.snip

```diff
@@ -13,20 +13,32 @@
 @snippet sampleCode_(apiMethod)
   require "{@apiMethod.initCode.apiFileName}"
 
+  {@sampleCodeAliases(apiMethod)}
+
   {@sampleCodeCore(apiMethod)}
 @end
 
-@snippet sampleCodeCore(apiMethod)
+@snippet sampleCodeAliases(apiMethod)
   @join fileImport : apiMethod.initCode.importSection.appImports
     @join alias : fileImport.types
       {@alias.nickname} = {@alias.fullName}
     @end
   @end
-  {@BREAK}
+@end
+
+@snippet sampleCodeCore(apiMethod)
   {@apiMethod.apiVariableName} = {@apiMethod.apiClassName}.new
   @if apiMethod.initCode.lines
     {@initCode(apiMethod.initCode)}
   @end
+  # The different *SampleCode(apiMethod) snippets include calls to
+  # non-declared methods, like:
+  #
+  # @# Process the response.
```

Comment:
The fix for smoke test is trivial, I already have it. It's very hard, if at all possible, to put these outside method_sample.snip, because of these samples:

```ruby
library_service_client = LibraryServiceClient.new

# Iterate over all results.
library_service_client.list_shelves.each do |element|
  # Process element.
  process_element element
end

# Or iterate over results one page at a time.
library_service_client.list_shelves.each_page do |page|
# Process each page at a time.
  page.each do |element|
    # Process element.
    process_element element
  end
end
```

```ruby
library_service_client = LibraryServiceClient.new

# Register a callback during the method call.
operation = library_service_client.get_big_book(formatted_name) do |op|
  raise op.results.message if op.error?
  op_results = op.results
  # Process the results.
  process_results op_results

  metadata = op.metadata
  # Process the metadata.
  process_metadata metadata
end

# Or use the return value to register a callback.
operation.on_done do |op|
  raise op.results.message if op.error?
  op_results = op.results
  # Process the results.
  process_results op_results

  metadata = op.metadata
  # Process the metadata.
  process_metadata metadata
end

# Manually reload the operation.
operation.reload!

# Or block until the operation completes, triggering callbacks on
# completion.
operation.wait_until_done!
```

Initially when I was only doing tests of basic unary calls, the verification check was being output by test.snip, just after the sample.

I'll be very happy to have a workaround or alternative, though!